conference.js

```diff
@@ -1928,5 +1929,18 @@ export default {
      */
     removeListener (eventName, listener) {
         eventEmitter.removeListener(eventName, listener);
+    },
+
+    /**
+     * Checks if the participant given by participantId is currently in the
+     * last N set if there's one supported.
+     *
+     * @param participantId the identifier of the participant
+     * @returns {boolean} {true} if the participant given by the participantId
+     * is currently in the last N set or if there's no last N set at this point
+     * and {false} otherwise
+     */
+    isInLastN (participantId) {
+        return room.isInLastN(participantId);
```

Comment:
This is currently handled by the library. JitsiConference.isInLastN will always return true if lastN is disabled. This what the JitsiConference.isInLastN jsdoc says:
" @return {boolean} true if the participant with id is in the last N set or
 + * if there's no last N set, false otherwise.
 + */"
 Do you mean that we should add it here as well?